public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/situation.ts

```diff
@@ -0,0 +1,526 @@
+import type { Tree, SyntaxNode } from '@lezer/common';
+
+import { parser } from '@grafana/lezer-logql';
+
+type Direction = 'parent' | 'firstChild' | 'lastChild' | 'nextSibling';
+type NodeTypeName =
```

Comment:
here instead of hardcoded values, we should use imported ids in the same way as we do in prometheus https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/components/monaco-query-field/monaco-completion-provider/situation.ts#L29